lib/WebpackError.js

```diff
@@ -0,0 +1,21 @@
+/*
+	MIT License http://www.opensource.org/licenses/mit-license.php
+	Author Jarid Margolin @jaridmargolin
+*/
+"use strict";
+
+module.exports = class WebpackError extends Error {
+	constructor() {
+		super();
+
+		if(this.initialize) {
+			this.initialize.apply(this, arguments);
```

Comment:
Just a general note: In general it's a bad practice to call a (virtual) function in the constructor of the base class. Why? Because it creates a weird call order of the methods that could lead to errors.

By default you usually assume in a class that the constructur runs before any other method. If you call a virtual method in the base class constructor this method (in the child class) runs before the child class constructor. That's weird and lead to problems if you want to use constructor arguments in the (virtual) method.

Note that it doesn't lead to errors in your case, but we should still apply good practices here. In addition to this it's easier to understand a constructor (even without reading the base class).